libs/NfcCoreLib/lib/Nci/phNciNfc.c

```diff
@@ -677,6 +677,70 @@ NFCSTATUS phNciNfc_Nfcee_ModeSet(void * pNciHandle,
     return wStatus;
 }
 
+NFCSTATUS phNciNfc_Nfcee_SePowerAndLinkCtrlSet(void * pNciHandle,
+                                               void * pNfceeHandle,
+                                               phNciNfc_PowerLinkModes_t eActivationMode,
+                                               pphNciNfc_IfNotificationCb_t pNotifyCb,
+                                               void *pContext)
+{
+    NFCSTATUS wStatus = NFCSTATUS_SUCCESS;
+    phNciNfc_Context_t * pNciContext = (phNciNfc_Context_t *)pNciHandle;
+    pphNciNfc_NfceeDeviceHandle_t pDevHandle = \
```

Comment:
nit: I am not sure why this file uses c-preprocessor newline escapes ('\\') everywhere. Though I wouldn't include them.

Reply:
I have noted a couple of newline escapes ('\') characters in the project. Not sure why neither.
I have removed the ones showing in that patch.